packages/in_app_purchase/in_app_purchase_android/android/src/main/java/io/flutter/plugins/inapppurchase/MethodCallHandlerImpl.java

```diff
@@ -65,10 +66,27 @@ static final class MethodNames {
     static final String IS_FEATURE_SUPPORTED = "BillingClient#isFeatureSupported(String)";
     static final String GET_CONNECTION_STATE = "BillingClient#getConnectionState()";
     static final String GET_BILLING_CONFIG = "BillingClient#getBillingConfig()";
+    static final String IS_ALTERNATIVE_BILLING_ONLY_AVAILABLE =
+        "BillingClient#isAlternativeBillingOnlyAvailable()";
+    static final String CREATE_ALTERNATIVE_BILLING_ONLY_REPORTING_DETAILS =
+        "BillingClient#createAlternativeBillingOnlyReportingDetails()";
+    static final String SHOW_ALTERNATIVE_BILLING_ONLY_INFORMATION_DIALOG =
+        "BillingClient#showAlternativeBillingOnlyInformationDialog()";
```

Comment:
For the record since I've missed this before: according to find in page, these three strings exactly match the Dart versions.

Reply:
Thanks for the double check!